progressbar/widgets.py

```diff
@@ -227,7 +227,7 @@ class Percentage(Widget):
     """Displays the current percentage as a number with a percent sign."""
 
     def update(self, pbar):
-        return '%3d%%' % pbar.percentage()
+        return '%3.0f%%' % pbar.percentage()
```

Comment:
Since `percentage()` returns a `float`, this is more correct in general, but it's most important when `maxval` is `UnknownLength` because `percentage()` returns `NaN`, which can't be formatted by `%d`